docs/reference/method/MongoDBModelCollectionInfo-getIdIndex.txt

```diff
@@ -49,7 +49,7 @@ Examples
 
    var_dump($info->getIdIndex());
 
-The output would then resemble::
+The output would then resemble:
```

Comment:
Is there a reason this doesn't have `.. code-block:: none` like most of the other occurrences?